Manuals/Tools/Altibase_trunk/eng/Migration Center User's Manual.md

```diff
@@ -902,43 +902,49 @@ The primary option is the Migration Type: either “DB to DB” or “DB to File
-| Data Validation Options : Data Sampling                                | Specifies whether or not to perform data validation on sampling data only.                                                                                                                                                                      |
-| Data Validation Options : Percent Sampling (exact counting)            | Specifies the percentage of data to be sampled from the table. This option is used only if the Exact Counting Method was selected in the Build step.                                                                                            |
-| Data Validation Options : Record Count Sampling (approximate counting) | Specifies the number of records to sample from the table. This option is used only if the Approximate Counting Method was selected in the Build step.                                                                                           |
+| Name                                         | Description                                                  |
+| -------------------------------------------- | ------------------------------------------------------------ |
+| Execution Thread                             | Specifies the maximum number of multi-thread to be used when executing data migration. The default value is triple of the number of logical processors in the server running the Migration Center. The suggested range of this value is 1 to triple of the number of logical processors in the server running the Migration Center. |
+| Migration Target                             | Specifies the targets for data migration: <br />\- Object & Data: Database objects and table data<br />\- Object: Database objects only |
+| **Object Options**                           |                                                              |
+| Foreign Key Migration                        | Specifies whether or not to include foreign key constraints in migration target. This option is disabled by default. |
+| PSM Migration                                | Specifies whether or not to include  PSM objects such as procedures, functions, materialized views, views, typesets, and triggers in migration target. This option is enabled by default.<br /> |
+| Drop Existing Objects                        | Specifies whether or not to recreate the database object before data migration. <br />When this option is enabled, DROP and CREATE will be executed on every objects in target database. When this option is disabled, data migration will be executed without dropping the database objects. This option is disabled by default. |
+| Keep Partition Table                         | Specifies whether or not to maintain partitioned tables.<br />When this option is enabled, partitioned tables will be created the same as the source database. When this option is disabled, it will be created as a non-partitioned table. This option is disabled by default. |
```

Comment:
When this option is enabled, partitioned tables will be created the same as the source database.
-> When this option is enabled, partitioned tables in the the source database will be migrated as partitioned tables if applicable.
파티션 테이블 종류와 타입에 따라 변환 가능한 경우만 파티션 테이블을 유지합니다. 

When this option is disabled, it will be created as a non-partitioned table.
-> When this option is disabled, target partitioned tables will be migrated as non-partitioned tables.
앞 문장과 일관성 있게 복수형을 쓰는 게 좋을 것 같습니다.

Reply:
When this option is enabled, partitioned tables in the the source database will be migrated as partitioned tables if applicable according to the type.
타입에 따라 결정된다는 것을 강조하기 위해 뒤에 추가하는 것은 어떤가요?

Reply:
> When this option is enabled, partitioned tables will be created the same as the source database. -> When this option is enabled, partitioned tables in the the source database will be migrated as partitioned tables if applicable. 파티션 테이블 종류와 타입에 따라 변환 가능한 경우만 파티션 테이블을 유지합니다.

한글 매뉴얼 문장이 이러한데요, 
Yes는 원본 데이터베이스와 동일하게 파티션드 테이블 생성한다.

아래와 같이 변경하고자 합니다. 
Yes는 파티션 테이블 종류와 타입에 따라 변환 가능한 경우 파티션드 테이블을 유지합니다.

Reply:
// 희정님
Yes는 파티션 테이블 종류와 타입에 따라 변환 가능한 경우 파티션드 테이블을 유지합니다. -> 고치신 문장이 더 명확해 보입니다. 감사합니다.

Reply:
// 혜인님,
타입에 따라 결정된다는 것을 강조하기 위해 뒤에 추가하는 것은 어떤가요?
-> 명확한 표현의 위해 고민해주셔서 감사합니다. 다만 이 부분은 DBMS vendor가 지원하는 파티션드 테이블 DDL 문법, 데이터 타입, 중첩 지원 여부 등 다양한 차이에 의해 변환 가능/불가능한 경우가 있어 타입때문이라고 딱 잘라 말하기는 힘들 것 같습니다.

Reply:
충헌님 설명 감사합니다. 희정님께서 작성해주신 문장 바탕으로 수정하겠습니다.

Reply:
질문이 있습니다.
국문의 경우 'Yes는 파티션 테이블 종류와 타입에 따라 변환 가능한 경우 파티션드 테이블을 유지합니다.'가 더 명확한 문장이라고 하셨는데, 테이블 종류와 타입이라는 말이 동의어가 아닌 건가요? 
또한, 영문에 타입에 따라 결정된다는 것을 추가할지 여쭤본 질문에 '이 부분은 DBMS vendor가 지원하는 파티션드 테이블 DDL 문법, 데이터 타입, 중첩 지원 여부 등 다양한 차이에 의해 변환 가능/불가능한 경우가 있어 타입때문이라고 딱 잘라 말하기는 힘들 것 같습니다.'라고 답변을 해주셨는데, 국문 문장의 '테이블 종류와 타입'이라는 단어에는 이 내용이 포함되어 있는 것인가요?

Reply:
한국어와 영어는 context sensitive에 대한 언어적인 차이가 있습니다. 예를 들어 우스개 소리로 한국어에서 '거시기가 거시기해서 거시기한다'라고 표현해도 context sensitive가 높기 때문에 문맥(context)을 아는 사람들은 이해합니다. 반면에 영어는 context sensitive가 낮기 때문에 이렇게 표현하기도 힘들고, 이런 방식으로 표현하면 이해가 되지 않습니다.

한국어로 'Yes는 파티션 테이블 종류와 타입에 따라 변환 가능한 경우 파티션드 테이블을 유지합니다.'라고 표현해도 '종류와 타입'이 많은 뜻을 내포 (두리뭉술)하기 때문에  이해에 무리가 없다고 생각합니다. 하지만, 영어로 'When this option is enabled, partitioned tables in the the source database will be migrated as partitioned tables if applicable according to the type.'라고 표현했을 때는 type이 partition table type만으로 이해가 되기 때문에, 타입 이외의 여러 차이를 나열했습니다.  여러 차이를 명확하게 나열하거나 모호함을 위해 아예 안 쓰는 것도 한가지 방법이라고 생각합니다.

저도 영어는 native가 아니기 때문에 꼭 옳다고는 할 수 없고, 다만 해당 표현에서 느끼는 부분을 말씀드린 겁니다. :) 
그리고, 제가 말씀드린 부분은 전체 문맥을 바꿀 만큼 크지 않기 때문에, 최종 판단은 희정님과 혜인님이 판단하시고 결정부탁드립니다.

감사합니다.

Reply:
충헌님, 정성어린 답변 감사합니다. 왜 차이가 있게 말씀하셨는지 바로 이해하였습니다.
희정님께 이 부분은 어떻게 하면 좋을지 의논해보겠습니다. 다시 한번 감사합니다. ^^

Reply:
충헌님과 논의해서.. 

Yes는 변환 가능한 경우 원본 데이터베이스와 동일한 파티션드 테이블을 생성한다. 이 경우 사용자는 조정(Reconcile) 단계에서 파티션드 테이블 변환에 필요한 추가 작업을 진행해야 한다. 

이렇게 정리하기로 했습니다. 혜인님과도 확인했습니다.